src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,69 @@
+@use "../utlis/variables";
+
+.stopwatch {
+  height: variables.$clock-size;
+  width: variables.$clock-size;
+  background-color: variables.$clock-face-color;
+  border-radius: variables.$clock-radius;
+  position: relative;
+
+  &__center {
+    height: variables.$clock-center-size;
+    width: variables.$clock-center-size;
+    background-color: variables.$clock-center-color;
+    top: calc(variables.$clock-size / 2) - (variables.$clock-center-size / 2);
+    left: calc(variables.$clock-size / 2) - (variables.$clock-center-size / 2);
+    border-radius: variables.$clock-radius;
+    position: absolute;
+  }
+
+  &__minutes {
+    height: variables.$minutes-length;
+    width: variables.$minutes-width;
+    position: absolute;
+    background-color: variables.$minutes-color;
+    left: calc(variables.$clock-size / 2) - (variables.$minutes-width / 2);
+    top: calc(variables.$clock-size / 2) - variables.$minutes-length;
+    border-radius: variables.$clock-arms-radius;
+    transform-origin: bottom;
+    animation-duration: 60s;
+    animation-name: minutes;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
+  }
+
+  &__seconds {
+    height: variables.$seconds-length;
+    width: variables.$seconds-width;
+    position: absolute;
+    background-color: variables.$seconds-color;
+    left: calc(variables.$clock-size / 2) - (variables.$seconds-width / 2);
+    top: calc(variables.$clock-size / 2) - variables.$seconds-length;
+    border-radius: variables.$clock-arms-radius;
+    transform-origin: 50% 100%;
+    animation-duration: 5s;
+    animation-name: seconds;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
+  }
+}
+
+
+
+@keyframes seconds {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes minutes {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
```suggestion

```
Consider using DRY principle and avoid repeating code, your keyframes look the same